src/client.rs

```diff
@@ -600,6 +690,91 @@ impl Client {
     }
 }
 
+/// [Actions](https://www.w3.org/TR/webdriver1/#actions)
+impl Client {
+    /// Perform the specified input actions.
+    ///
+    /// ```ignore
+    /// use fantoccini::actions::ActionChain;
+    ///
+    /// let chain = ActionChain::builder()
+    ///     .click_element(element)
+    ///     .pause(Duration::from_secs(2))
+    ///     .send_keys("fantoccini")
+    ///     .build();
+    /// client.perform_actions(chain).await?;
+    /// ```
+    ///
+    /// See [17.5 Perform Actions](https://www.w3.org/TR/webdriver1/#perform-actions) of the
+    /// WebDriver standard.
+    #[cfg_attr(docsrs, doc(alias = "Perform Actions"))]
+    pub async fn perform_actions(&mut self, actions: ActionChain) -> Result<(), error::CmdError> {
```

Comment:
What do you think of adapting this slightly to have
```rust
impl Client {
  fn action_chain(&mut self) -> ActionChain;
}
impl ActionChain {
  fn perform(self) -> Result<(), error::CmdError>;
}
```
(where `ActionChain` here is the same as the builder type you currently have). I _think_ that'd make for a nicer interface?

Reply:
I agree, but currently the builder isn't really capable of covering everything that the `Actions` spec supports. It doesn't handle touch/pen actions, multiple of the same type, or chords. However I can probably just add builder methods to allow the user to pass in an entire channel, which should fill the gap.

This does conflict with your other suggestion to move `ActionChainBuilder` into `thirtyfour`, which I think might be the better option but I don't want to leave the `fantoccini` side feeling too low-level to be useful.

Reply:
Yeah, I think I changed my mind on this — I think we should leave the builder out of `fantoccini`. As we talked about earlier, `fantoccini` _should_ be a low-level interface, and an action builder feels antithetical to that. It also might mean that we can more easily add support for the bits of actions that aren't yet supported, since we don't _also_ have to come up with a builder interface for it.

Reply:
So would you leave `Client::perform_actions()` as is then?